tests/main/interfaces-network-bind/task.yaml

```diff
@@ -29,6 +29,15 @@ prepare: |
 
     EOF
 
+restore: |
+    # We had a runaway network-bind-consumer process that would generate
```

Comment:
I'm -1 on such ad hoc change. Either we have a broken system where we leak snap processes around and we need to fix that or we need to make all tests remove their snaps explicitly. It cannot be "just here" really.

Reply:
Yeah, that's what I mean. This can also happen in any number of other tests so fixing it here feels wrong.

Reply:
Perhaps the alternative is to use cursors instead of rotating the logs. I started a branch a time ago, I could make some progress on that this week.

Reply:
Unfortunately (while a perfect solution) we cannot rely on cursors in the oldest build of systemd that we support.

Reply:
@zyga @mvo5 the PR #5151 implements the change we need to avoid this issue.